ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/documentation/devices/DC1-SPINE1.md

```diff
@@ -447,6 +447,11 @@ ip route vrf MGMT 0.0.0.0/0 192.168.200.5
 
 ### Router BGP EVPN Address Family
 
+#### EVPN Host Flapping Settings
```

Comment:
Should not be generated for devices where it is not configured.
Problem is that documentation looks at wrong variables.